stacks_queues/stack.py

```diff
@@ -12,7 +12,8 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.stack_size += 1 
+        self.stack_list.append(element)
```

Comment:
Again, `stack_list` and `append` don't exist as attributes/methods for the `LinkedList` class we provide